go/chat/inboxsource.go

```diff
@@ -16,13 +16,17 @@ import (
 
 type Inbox struct {
 	Version    chat1.InboxVers
-	Convs      []chat1.ConversationLocal
+	Convs      []chat1.ConversationLocalWithBreaks
 	Pagination *chat1.Pagination
```

Comment:
In other words, could we just have an additional field here for identify failures?